src/resonanceReconstruction/rmatrix/src/coh3-coulomb.hpp

```diff
@@ -0,0 +1,39 @@
+const double HBAR       =  6.582119514e-22; /* Planck's constant/2pi [MeV sec]*/
+const double VLIGHT     =  2.99792458e+23 ; /* light velocty [fm/sec]         */
```

Comment:
The hbar here is CODATA2014, as is the one in resonanceReconstruction.hpp so it's consistent.

Reply:
The physical constants given here (hbar, c, elementary charge, amu to energy conversion, permittivity) do not appear to be used in the code since I provide eta in the coulomb wave function call. We can probably remove it.